src/components/molecules/Human/config.ts

```diff
@@ -0,0 +1,10 @@
+export const categories = [
```

Comment:
This belongs somewhere in `src/app` or `src/containers`. `src/containers` could serve as a "smart" middle-man between routing (`src/app`) and UI (`src/components`).
Focus on keeping those layers separate.